src/components/artwork_filter/dropdown.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import Icon from '../icon'
+
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import { primary, secondary } from "../../assets/fonts"
+
+export interface DropdownProps extends RelayProps, React.HTMLProps<Dropdown> {
+  aggregation: any
+}
+
+export interface DropdownState {
+  is_hovered: boolean
+}
+
+export class Dropdown extends React.Component<DropdownProps, DropdownState> {
+  constructor(props) {
+    super(props)
+    this.state = {
+      is_hovered: false
+    }
+  }
+  
+  toggleHover() {
+    this.setState({
+      is_hovered: !this.state.is_hovered
+    })
+  }
+
+  render() {
+    const navItems = this.props.aggregation.counts.map((count) =>{
+      return (
+        <NavItem>
+          <span>{count.name}</span>
+          <span> ({count.count})</span>
+        </NavItem>
+      )
+    })
+
+    let buttonColor = "white"
+    let buttonTextColor = "black"
+    let navStyle = { display: "none" }
+
+    if (this.state.is_hovered) {
+      buttonColor = "black"
+      buttonTextColor = "white"
+      navStyle = { display: "block" }
+    }
```

Comment:
Good question, I couldn't figure out a non-complicated way of doing this with `styled-components`. If you can think of a more straightforward way to do it, lmk!